### PR TITLE
fix for CI build errors

### PR DESCRIPTION
**Pull request by deepakrkris**
errors in CI builds

### CODE REVIEW

File: src/parser/typeRegistry.js [hunk 1]
```diff
@@ -52,7 +52,7 @@ var elementTypes = [
   './xsd/extension',
   './xsd/group',
   './xsd/import',
-  './xsd/include',
+  './xsd/Include',
```
Comment (raymondfeng) [inline src/parser/typeRegistry.js]: Why this change? If the file name is wrong, we should fix it.

Comment (deepakrkris) [inline src/parser/typeRegistry.js]: @raymondfeng the file name is not wrong, it has a capital "I" ; we can either change the file name or the reference to it. If you think the standard is to have file names with small case letters, I will do that

Comment (raymondfeng) [inline src/parser/typeRegistry.js]: We should rename the file to be `include.js`.

Comment (deepakrkris) [inline src/parser/typeRegistry.js]: @raymondfeng done. Do I have a +1 ?

**Comment by deepakrkris** [comment]
@slnode test please
